Gemfile

```diff
@@ -15,6 +15,7 @@ group :oracle do
   gem "ruby-oci8"
   gem "activerecord-oracle_enhanced-adapter"
 end
+gem "where-or"
```

Comment:
Should we look into where we can use this other places? (can definitely be separate PRs, and we should figure out how much effort we want to spend on it)

Reply:
Sure, I'll add a ticket to Rails 5 upgrade land.